FrameworkSystem/private/monitoring/MonitoringCatalog.py

```diff
@@ -50,7 +58,7 @@ def __dbExecute(self, query, values=False):
         executed = True
       except Exception as e:
         self.log.exception("Exception executing statement", "query: %s, values: %s" % (query, values))
-        time.sleep(0.01)
+        time.sleep(random.random())
```

Comment:
I think I can guess but maybe add a comment explaining why a random pause is being used?